projects/ui-components/lf-login/lf-login.component.ts

```diff
@@ -184,6 +197,11 @@ export class LfLoginComponent implements OnChanges, OnDestroy {
   @Output() loginCompleted = new EventEmitter<void>();
   @Output() logoutCompleted = new EventEmitter<AbortedLoginError | void>();
 
+/**
+ * Refreshes and returns the access token
+ * @param initiateLoginFlowOnRefreshFailure
```

Comment:
add descriptions for initiateLoginFlowOnRefreshFailure
"If true, attempts to login using OAuth flow if refresh fails (follows same behavior as clicking the button)"